projects/1.18/assets/create-clockwork/vs_clockwork/lang/zh_cn.json

```
@@ -0,0 +1,129 @@
+{
+  "itemGroup.clockwork": "VS：发条装置",
+  "itemGroup.vs_clockwork": "VS: 发条装置",
```

```suggestion
  "itemGroup.vs_clockwork": "VS：发条装置",
```
还是标点问题
**这类问题请全文通查并修正**